app/views/pafs_core/projects/index.html.erb

```diff
@@ -38,7 +38,7 @@
             <th scope="col" class="sortable-heading">Project name</th>
             <th scope="col">Created by</th>
             <th scope="col">
-              <% sort_properties_for_col = get_next_sort_order_and_curr_arrow(@project_sort_column, "updated_at", @project_sort_order, "updated_at", "desc") %>
```

Comment:
Could we move this into another PR?

We could do this as one piece of house keeping and keep the changes contained to one PR :)

Reply:
Not really - this is part of the fix :)

Reply:
Cool.